setup/hotspotServerSetup

```diff
@@ -43,12 +43,15 @@ sudo systemctl disable dhcpcd
 echo "Setting up Flask, uWSGI, and NGINX"
 sudo export FLASK_DEBUG=1
 sudo mkdir /home/pi/solarSENSE
+sudo mkdir /home/pi/solarSENSE/app
 sudo chown www-data /home/pi/solarSENSE
-sudo cp ./home.py /home/pi/solarSENSE
-sudo mkdir /home/pi/solarSENSE/static
-sudo mkdir /home/pi/solarSENSE/templates
-sudo cp ./static/solarspell.jpg /home/pi/solarSENSE/static
-sudo cp ./templates/index.html /home/pi/solarSENSE/templates
+sudo cp ./run.py /home/pi/solarSENSE
+sudo cp ./config.py /home/pi/solarSENSE
+# sudo mkdir /home/pi/solarSENSE/static
```

Comment:
Go ahead a delete these commented out lines

Reply:
Got this done sorry took some time, was also working on adding some other things as well